.env

```
@@ -0,0 +1,32 @@
+# Migration selection
+RP_TOKEN_MIGRATION=false
+RP_SINGLEBUCKET_MIGRATION=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RP_SINGLEBUCKET_MIGRATION key should go before the RP_TOKEN_MIGRATION key

(UnorderedKey)